tests/robustness/snapstore/simple.go

```diff
@@ -0,0 +1,71 @@
+package snapstore
+
+import "sync"
+
+var _ Storer = &Simple{}
+
+// Simple is a snapstore implementation that stores
+// snapshot metadata as a byte slice in a map in memory
+type Simple struct {
+	s map[string][]byte
+	l *sync.Mutex
+}
+
+// NewSimple instantiates a new Simple snapstore and
+// returns its pointer
+func NewSimple() *Simple {
+	return &Simple{
+		s: make(map[string][]byte),
+		l: new(sync.Mutex),
+	}
+}
+
+// Store implements the Storer interface Store method
+func (store *Simple) Store(key string, val []byte) error {
```

Comment:
It makes sense to have single-letter (e.g., `s`) or short names for receivers. Perhaps rename the `s` field to `m`?